packages/schematics/angular/application/index_spec.ts

```diff
@@ -16,108 +17,92 @@ describe('Application Schematic', () => {
-    expect(files.indexOf('/foo/.editorconfig')).toBeGreaterThanOrEqual(0);
-    expect(files.indexOf('/foo/.angular-cli.json')).toBeGreaterThanOrEqual(0);
-    expect(files.indexOf('/foo/.gitignore')).toBeGreaterThanOrEqual(0);
-    expect(files.indexOf('/foo/karma.conf.js')).toBeGreaterThanOrEqual(0);
-    expect(files.indexOf('/foo/package.json')).toBeGreaterThanOrEqual(0);
-    expect(files.indexOf('/foo/protractor.conf.js')).toBeGreaterThanOrEqual(0);
-    expect(files.indexOf('/foo/README.md')).toBeGreaterThanOrEqual(0);
-    expect(files.indexOf('/foo/tsconfig.json')).toBeGreaterThanOrEqual(0);
-    expect(files.indexOf('/foo/tslint.json')).toBeGreaterThanOrEqual(0);
-    expect(files.indexOf('/foo/e2e/app.e2e-spec.ts')).toBeGreaterThanOrEqual(0);
-    expect(files.indexOf('/foo/e2e/app.po.ts')).toBeGreaterThanOrEqual(0);
-    expect(files.indexOf('/foo/src/favicon.ico')).toBeGreaterThanOrEqual(0);
-    expect(files.indexOf('/foo/src/index.html')).toBeGreaterThanOrEqual(0);
-    expect(files.indexOf('/foo/src/main.ts')).toBeGreaterThanOrEqual(0);
-    expect(files.indexOf('/foo/src/polyfills.ts')).toBeGreaterThanOrEqual(0);
-    expect(files.indexOf('/foo/src/styles.css')).toBeGreaterThanOrEqual(0);
-    expect(files.indexOf('/foo/src/test.ts')).toBeGreaterThanOrEqual(0);
-    expect(files.indexOf('/foo/src/tsconfig.app.json')).toBeGreaterThanOrEqual(0);
-    expect(files.indexOf('/foo/src/tsconfig.spec.json')).toBeGreaterThanOrEqual(0);
-    expect(files.indexOf('/foo/src/typings.d.ts')).toBeGreaterThanOrEqual(0);
-    expect(files.indexOf('/foo/src/assets/.gitkeep')).toBeGreaterThanOrEqual(0);
-    expect(files.indexOf('/foo/src/environments/environment.prod.ts')).toBeGreaterThanOrEqual(0);
-    expect(files.indexOf('/foo/src/environments/environment.ts')).toBeGreaterThanOrEqual(0);
-    expect(files.indexOf('/foo/src/app/app.module.ts')).toBeGreaterThanOrEqual(0);
-    expect(files.indexOf('/foo/src/app/app.component.css')).toBeGreaterThanOrEqual(0);
-    expect(files.indexOf('/foo/src/app/app.component.html')).toBeGreaterThanOrEqual(0);
-    expect(files.indexOf('/foo/src/app/app.component.spec.ts')).toBeGreaterThanOrEqual(0);
-    expect(files.indexOf('/foo/src/app/app.component.ts')).toBeGreaterThanOrEqual(0);
+    expect(files.indexOf('/projects/foo/environments/environment.prod.ts'))
+      .toBeGreaterThanOrEqual(0);
+    expect(files.indexOf('/projects/foo/environments/environment.ts')).toBeGreaterThanOrEqual(0);
+    expect(files.indexOf('/projects/foo/favicon.ico')).toBeGreaterThanOrEqual(0);
+    expect(files.indexOf('/projects/foo/index.html')).toBeGreaterThanOrEqual(0);
+    expect(files.indexOf('/projects/foo/karma.conf.js')).toBeGreaterThanOrEqual(0);
+    expect(files.indexOf('/projects/foo/main.ts')).toBeGreaterThanOrEqual(0);
+    expect(files.indexOf('/projects/foo/polyfills.ts')).toBeGreaterThanOrEqual(0);
+    expect(files.indexOf('/projects/foo/protractor.conf.js')).toBeGreaterThanOrEqual(0);
```

Comment:
`protractor.conf.js` is a configuration file for the `e2e` project. You can find the intended structure here: https://github.com/filipesilva/workspace-project/tree/master/projects/app-e2e

I think we should just call it `e2e` instead of `app-e2e` though. It's cleaner and if users want more e2e project they can rename it.